x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_details.ts

```diff
@@ -51,6 +51,7 @@ export function useFetchSloDetails({
           return response;
         } catch (error) {
           // ignore error for retrieving slos
+          return null;
```

Comment:
Returning `null` here because ReactQuery doesn't like returning `undefined` and we are relying on the `404` exception in `useFetchSloOrCompositeSloDetails`.